overview/api/index.md

```diff
@@ -27,7 +27,8 @@ The main APIs for our components are:
 | [Provisioning API](http://docs.provisioningapi.apiary.io/) | Provisioning is a service creating accounts for [sandboxes](https://help.keboola.com/manipulation/transformations/sandbox/), [transformations](https://help.keboola.com/manipulation/transformations/) and database writers. | [APIB](https://github.com/keboola/provisioning-client/blob/master/apiary.apib)
 | [Provisioning Management API](http://docs.provisioningmanagementapi.apiary.io/) | API managing servers for [sandboxes](https://help.keboola.com/manipulation/transformations/sandbox/), [transformations](https://help.keboola.com/manipulation/transformations/). | [APIB](https://github.com/keboola/provisioning-bundle/blob/master/apiary.apib)
 | [Syrup Queue API](http://docs.syrupqueue.apiary.io/#) | Syrup Queue is a component managing [Jobs](/integrate/jobs/). | [APIB](https://github.com/keboola/syrup-queue/blob/master/apiary.apib)
-| [OAuth Manager API](http://docs.oauthv2.apiary.io/) | OAuth2 is a component managing [OAuth authorizations](/extend/common-interface/oauth/#authorize) of other components. | [APIB](https://github.com/keboola/oauth-v2-bundle/blob/master/apiary.apib)
+| [OAuth Manager API v2](http://docs.oauthv2.apiary.io/) | OAuth Manager (v2) is a component managing [OAuth authorizations](/extend/common-interface/oauth/#authorize) of other components. | [APIB](https://github.com/keboola/oauth-v2-bundle/blob/master/apiary.apib)

```

Comment:
prijde mi to trosku zmateny s tema verzema a nazvama oauthv2 se to myslim jmenovalo kvuli oauth2, nadepsany to tady je oboji OAuth Manager, ale tady https://oauthapi3.docs.apiary.io/# je jen OAuth API. Radsi nez OAuth Manager v3 bych to nazval nejak jinak (OAuth Broker? nevim) at nam tam nafurt nestrasi ta 3 (kdyz to spolu nema v podsate nic spolecnyho)

a k tomu starymu bych kazdopadne napsal, ze je deprecated (i do apib) a do apib pridal odkaz na novy

Reply:
JJ OAuth Broker sa mi paci tiez. To sam tam chcel dat povodne. 

Ok, takze stary deprekovat a novy nazvat OAuth Broker.

Reply:
A to apib upravim ako pises